src/lambdas/GetInstanceUtilization/index.mjs

```diff
@@ -8,24 +8,25 @@ function naiveRound (num, decimalPlaces = 0) {
   return Math.round(num * p) / p
 }
 
-async function getInstanceUtilization (instanceId, region) {
+async function getInstanceUtilization (id, region, Namespace, MetricName, DimensionName) {
   const client = new CloudWatchClient({ region })
   const today = new Date()
   today.setHours(0, 0, 0, 0)
-  const yesterday = new Date(new Date().setDate(new Date().getDate() - 1))
+  const yesterday = new Date()
+  yesterday.setDate(today.getDate() - 1)
   yesterday.setHours(0, 0, 0, 0)
 
   const params = {
     StartTime: yesterday,
     EndTime: today,
-    MetricName: 'CPUUtilization',
-    Namespace: 'AWS/EC2',
+    MetricName,
+    Namespace,
     Period: 60 * 60 * 24, // one day
     Statistics: ['Average'],
     Dimensions: [
       {
-        Name: 'InstanceId',
-        Value: instanceId
+        Name: DimensionName,
```

Comment:
I suggest changing `DimensionName` to `dimensionName` to follow  javascript variable naming best practices.